components/common/banner.js

```diff
@@ -29,7 +29,6 @@ const Banner = ({ icon, children, onCloseClick }) => {
       {children}
       <CloseButton
         onClick={onCloseClick}
-        border='2px solid'
```

Comment:
i dont care either way on this - deferring to you

Reply:
i like it without the border :D

Reply:
I left it without the border for now since we're split 2-2 😬